package.json

```diff
@@ -17,5 +17,8 @@
     "vite": "^3.1.0",
     "vite-plugin-css-injected-by-js": "2.1.1"
   },
-  "license": "MIT"
+  "license": "MIT",
+  "dependencies": {
+    "@notionhq/client": "^2.2.2"
```

Comment:
I don't understand. It's package for notion
I send data to notion with using this official dep.

Reply:
`dependencies` — list of packages required to run published package
`devDependencies` — list of packages required to build local package

for example. this package required Vite only for build stage. same for the @notionhq/client package. after build step is done you have a single bundle script ready to be shipped. this bundle does not have the required dependencies because all the needs are already inside it. this way deps should be empty. if a package has any deps, they will be installed automatically while installing step to allow package work.

for example lets check @notionhq/client deps and devdeps:

![image](https://user-images.githubusercontent.com/15259299/206239222-58590145-e11c-4a8a-ac78-fa1b83a2083f.png)


### how to install package correctly

in package.json there are two sections for required packages: dependencies and devDependencies. when you install any package (by default as dep) you can add a flag to install it as devdep.

`npm install <package>`
`npm install --save-dev <package>`

`yarn add <package>`
`yarn add -D <package>`

Read [Specifying dependencies and devDependencies in a package.json file](https://docs.npmjs.com/specifying-dependencies-and-devdependencies-in-a-package-json-file)

Reply:
it’s not a dev deps because we need this dep in production for notion connection. I don’t understand what you want. Yarn.dev is correct.

Reply:
I moved to devDeps. Is is okay?